.github/workflows/pr-review-companion.yml

```diff
@@ -76,10 +77,10 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ~/.local
-          key: dotlocal-${{ runner.os }}-${{ hashFiles('.github/workflows/pr-review-companion.yml') }}
+          key: dotlocal-${{ runner.os }}-${{ hashFiles('.github/workflows/pr-review-companion.yml') }}-${{ steps.setup-python.outputs.python-version }}
```

Comment:
I'm not sure about `hashFiles('.github/workflows/pr-review-companion.yml')` it seems to be an empty string.

Reply:
Yeah, we don't checkout the repo in this workflow, so there's no file to cache - I think that's ok, we can increment a number at the end of the cache key if we make any breaking changes. So, can you remove this part?

Reply:
We can remove it from the venv cache key below, as well. Perhaps we also want to include the python version in there, I don't think the poetry lockfile includes it.

Reply:
thanks for answering this, I've add a trailing `0` with a comment to explain it.